packages/cli/src/package.ts

```diff
@@ -350,6 +350,7 @@ type CanonicalExportsFieldConfig =
   | {
       envConditions: Set<"worker" | "browser">;
       extra: Record<string, JSONValue>;
+      useMjsProxy: boolean;
```

Comment:
quite frankly... I would just make this the default and remove the config. I introduced this with a config to limit the scope of the change - both for consumers (as you might want to preserve the existing behavior) and for the test suite (I didn't want to update existing tests just yet). 

I think that the previous behavior was already *broken* and I don't think of it as of a correct behavior - so I would personally treat this as a bug fix. It's subjective though 🤷‍♂️

Reply:
Yeah, I'm planning to do a major soon to enable `exports` by default and the Rollup 3 upgrade and we can change the default for this in that. I'd probably want to keep the config though

Reply:
Fine by me 👍

Reply:
Also, on naming, how about `unwrappedDefaultExportInStrictESM` or something like that to indicate what it does rather than how it does it?

Reply:
changed the name to the suggested one